village/src/components/SmurfForm.js

```diff
@@ -25,6 +27,17 @@ class SmurfForm extends Component {
     this.setState({ [e.target.name]: e.target.value });
   };
 
+  addSmurf = (event, newSmurf) => {
+    event.preventDefault();// explicit
```

Comment:
While this does add a smurf to the database, it requires a page refresh for the user to see any changes. This isn't how single page applications should work. Please take some time to write this in such a way that the state that is used to render the smurf list is updated with the response of the Axios Post.